packages/entities/entities-vaults/src/components/VaultForm.vue

```diff
@@ -371,6 +372,41 @@
                 :label="t('form.config.hcv.fields.approle_response_wrapping.label')"
               />
             </div>
+            <div
+              v-else-if="configFields[VaultProviders.HCV].auth_method === VaultAuthMethods.CERT"
+              class="vault-form-config-auth-method-container"
+            >
+              <KInput
+                v-model.trim="configFields[VaultProviders.HCV].cert_auth_role_name"
+                autocomplete="off"
+                data-testid="vault-form-config-hcv-cert_auth_role_name"
+                :label="t('form.config.hcv.fields.cert_auth_role_name.label')"
+                :readonly="form.isReadonly"
+                required
+                type="text"
```

Comment:
I think we can remove line L#386 since the component `KInput` defaults to `type="text"`